Update readme templates and generation (#94)
Shopify/argo-admin-template: README.md

# Shopify Admin, app extensions - Starter code

This repo contains starter code for apps that use _Argo_ to extend the UI of the _Shopify Admin_.       
For more about _Argo_ and admin extensions, please [read our docs](https://shopify.dev/tutorials/product-subscription-extension-overview).

> ## Getting started

Reviewer: We should encourage people to use the CLI before manually cloning this repo: https://shopify.dev/tutorials/getting-started-product-subscription-extension

Clone this repo

```sh
git clone --depth=1 git@github.com:Shopify/argo-admin-template.git YOUR_EXTENSION_NAME
```

Generate starter code

```sh
cd YOUR_EXTENSION_NAME

# With npm
npm install
npm run generate -- --type=PRODUCT_SUBSCRIPTION

# With yarn
yarn
yarn generate --type=PRODUCT_SUBSCRIPTION
```

_**Note:** Currently `PRODUCT_SUBSCRIPTION` is the only supported extension type._

## What's next?

> Check out the generated readme and index files (at the root of this repo), they contain further instructions and code to get you started.

Reviewer: ```suggestion
The `generate` command has created new files at the root of this repo. Check out the new `README.md` for further instructions, and `index.js/ts/tsx` for starter code.```

# Shopify Admin, app extensions - Starter code

This repo contains starter code for apps that use _Argo_ to extend the UI of the _Shopify Admin_.       

For more about _Argo_ and admin extensions, please [read our docs](https://shopify.dev/tutorials/product-subscription-extension-overview).

## Getting started

[Use the Shopify App CLI to create your _Argo_-enabled extension](https://shopify.dev/tutorials/getting-started-product-subscription-extension#scaffold-a-product-subscription-app-extension).

> **Note:** There is no need for you to clone this repo directly.
> The CLI will take care of cloning the repo and generating all required files.

Reviewer: ```suggestion
**Note:** We recommend [using the Shopify App CLI to create your Argo-enabled extension](https://shopify.dev/tutorials/getting-started-product-subscription-extension#scaffold-a-product-subscription-app-extension).

If you clone this repo directly and follow the instructions below, you’ll need to set up the Shopify App CLI manually to use your extension.
```

> ## Contributing

Reviewer: Not sure about this one, do we expect 3p contributions here?

We also seem to have lost the 'Getting started' heading.
